hack/patches/018-remove-deployment-failure-post-upgrade-test.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/vendor/knative.dev/serving/test/upgrade/postupgrade.go b/vendor/knative.dev/serving/test/upgrade/postupgrade.go
+index 61f5b1121..bf06e493f 100644
+--- a/vendor/knative.dev/serving/test/upgrade/postupgrade.go
++++ b/vendor/knative.dev/serving/test/upgrade/postupgrade.go
+@@ -42,7 +42,8 @@ func ServingPostUpgradeTests() []pkgupgrade.Operation {
+ 		CreateNewServicePostUpgradeTest(),
+ 		InitialScalePostUpgradeTest(),
+ 		CRDStoredVersionPostUpgradeTest(),
+-		DeploymentFailurePostUpgrade(),
++// Removing DeploymentFailurePostUpgrade from SO 1.33 upgrade tests, as it won't work if the https://github.com/knative/serving/pull/14795 fix is not in SO 1.32
++//		DeploymentFailurePostUpgrade(),
+ 	}
+ }
+ 
+diff --git a/vendor/knative.dev/serving/test/upgrade/preupgrade.go b/vendor/knative.dev/serving/test/upgrade/preupgrade.go
+index d00f5ef86..367f79511 100644
+--- a/vendor/knative.dev/serving/test/upgrade/preupgrade.go
++++ b/vendor/knative.dev/serving/test/upgrade/preupgrade.go
+@@ -35,7 +35,8 @@ func ServingPreUpgradeTests() []pkgupgrade.Operation {
+ 		ServicePreUpgradeAndScaleToZeroTest(),
+ 		BYORevisionPreUpgradeTest(),
+ 		InitialScalePreUpgradeTest(),
+-		DeploymentFailurePreUpgrade(),
++// Removing DeploymentFailurePreUpgrade from SO 1.33 upgrade tests, as it won't work if the https://github.com/knative/serving/pull/14795 fix is not in SO 1.32
```

Comment:
@maschmid this seems not true afaik as the upgrade from 1.11 to 1.12 worked upstream and upgrade tests passed. The change that was fixed upstream was in 1.12 not 1.11, thus 1.11 should be fine.

Reply:
Hmm, the test https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift-knative_serverless-operator/2578/pull-ci-openshift-knative-serverless-operator-main-415-test-upgrade-aws-415/1772592938325381120  failed on "Service did not transition to Ready=False" , we don't know if the other assertion of "Traffic should still work" would work... I wonder if it would be expected in 1.11 to stay in Ready and not transition to False (because the deployment reason is not propagated at all?  (we could perhaps try a version of the test that only has the other assertion just to see how that behaves... )

Reply:
That tests needs to stay here and also work :)

Reply:
ack, dropping the patch then.